static/css/style.css

```diff
@@ -91,20 +91,30 @@ body, input {
 
 /* Script info box */
 #script-box {
+  padding-right: 1rem;
+  margin-left: auto;
   margin-right: 1rem;
-  margin-left: 1rem;
-  text-align: right;
-  width: 100%;
+  justify-content: center;
 }
 
 #script-name {
-  font-size: 2rem;
+  font-size: 1.75rem;
+  border: #808080 1px solid;
   color: #1a354c;
   overflow: hidden;
-  max-height: 90px;
+  width: 100%;
+  margin-right: 1rem;
+  min-width: 10rem;
+  max-width: 15rem;
+  background-color: white;
+  line-height: 2.5rem;
+  max-height: 2.5rem;
+  padding: 0 0.5rem;
+  border-radius: 0.5rem;
 }
 
 #script-description {
+  display: none; /* TODO: FIXME: remove this field */
```

Comment:
If we are removing this box from the UX we should probably just remove all the associated code with it.
We should only do that after both repos are synced, so opened https://github.com/bbcmicrobit/PythonEditor/issues/83.